source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts

```diff
@@ -10,9 +11,18 @@ export const defaultMicrosoftHosts: containers.HostContainer[] = [
 	new containers.OutlookContainer("Outlook Actionable Messages", "containers/outlook-container.css"),
 	new containers.DarkTeamsContainer("Microsoft Teams - Dark", "containers/teams-container-dark.css"),
 	new containers.LightTeamsContainer("Microsoft Teams - Light", "containers/teams-container-light.css"),
-	new containers.VivaConnectionsContainer("Viva Connections (Preview)", "containers/viva-connections-container.css"),
```

Comment:
Is these change expected?

Reply:
Yes viva connections is no longer in preview. The line below adds the same line without preview in name.